src/test/java/com/exasol/adapter/document/documentfetcher/files/AbstractFilesDocumentFetcherTest.java

```diff
@@ -30,10 +30,10 @@ void testSourceReferences() {
         when(fileLoader.loadFiles()).thenReturn(Stream.of(mockLoadedFile("file-1"), mockLoadedFile("file-2")));
         final FileLoaderFactory loaderFactory = mock(FileLoaderFactory.class);
         when(loaderFactory.getLoader(Mockito.any(), Mockito.any(), Mockito.any())).thenReturn(fileLoader);
-        final AbstractFilesDocumentFetcher<Object> documentFetcher = mock(AbstractFilesDocumentFetcher.class,
+        final AbstractFilesDocumentFetcher documentFetcher = mock(AbstractFilesDocumentFetcher.class,
```

Comment:
Maybe if you use class fields with Mock annotations these two methods would be easier to read

Reply:
yes. I refactored it. What do you think?